platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -41,22 +41,8 @@
 
 RJ45_TYPE = "RJ45"
 
-DMI_FILE = '/sys/firmware/dmi/entries/2-0/raw'
-DMI_HEADER_LEN = 15
-DMI_PRODUCT_NAME = "Product Name"
-DMI_MANUFACTURER = "Manufacturer"
-DMI_VERSION = "Version"
-DMI_SERIAL = "Serial Number"
-DMI_ASSET_TAG = "Asset Tag"
-DMI_LOC = "Location In Chassis"
-DMI_TABLE_MAP = {
-                    DMI_PRODUCT_NAME: 0,
-                    DMI_MANUFACTURER: 1,
-                    DMI_VERSION: 2,
-                    DMI_SERIAL: 3,
-                    DMI_ASSET_TAG: 4,
-                    DMI_LOC: 5
-                }
+VPD_DATA_FILE = "/var/run/hw-management/eeprom/vpd_data"
```

Comment:
Need to make sure that the file is available for the current version of HW MNG and above. Please check.